src/lib/core/ErrorStr.h

```diff
@@ -48,7 +48,15 @@ struct ErrorFormatter
     ErrorFormatter * Next;
 };
 
+struct ErrorStrStorage
+{
+    uint16_t Size() { return sizeof(buff); }
```

Comment:
```suggestion
    constexpr static uint16_t Size() { return sizeof(buff); }
```

or do we really want this as a method? Why is it not a `constexpr uint16_t kBufferSize = sizeof(buff)` ?